### DISCUSSION TITLE
Winners' certificates

### DISCUSSION THREAD

**Discussion by xuchenApriLie** [snap-stanford/ogb#409] [General]
I would like to know if the winner of the OGB-LSC will be awarded a certificate of merit. It can be in electronic form so that we can print it ourselves without the shipping efforts.

**Comment by weihua916** [comment]
Hi, thanks for your comment. Yes, let me ask the competition chair and get back to you.

**Reply by weihua916** [reply]
I just sent you the certificate. Let me know if you received it.
